models/commands.py

```diff
@@ -26,4 +27,26 @@ def get_commands(cls):
     @classmethod
     def is_valid(cls, command: str):
         """Checks if the given command is valid."""
-        return command in cls.get_commands()
\ No newline at end of file
+        return command in cls.get_commands()
+
+class CommandsDescription(Enum):
+    """Enum class for the commands that the user can use in the application."""
+    HELLO = "'hello' - It's a pleasure to hear it!"
+    ADD = "'add' - Add new contact to contacts. Usage: add <name>, <phone> (can be added several phone numbers)"
+    ADD_BIRTHDAY = "'add_birthday' - Add bithday date to existed contact. Usage: add_birthday <name> <birthday> (dd.mm.yyyy)"
+    SHOW_BIRTHDAY = "'show_birthday' - Show birthday for selected contact. Usage: show_birthday <name>"
+    BIRTHDAYS = "'birthdays' - Show all upcoming birthdays in next 7 days."
+    CHANGE = "'change' - Change phone number for existed contact. Usage: change <name> <phone> <new_phone>"
+    PHONE = "'phone' - Show phone number(s) for selected contact. Usage: phone <name>"
+    ALL = "'all' - Show all existed contacts with data"
+    EXIT = "'exit' - Save data and close the bot"
+    CLOSE = "'close' - Save data and close the bot"
+    HELP = "'help' - Show info about existed command for the bot"
+
+    def __str__(self):
+        return self.value
+
+    @classmethod
+    def get_commands_description(cls):
+        """Returns a list of all existed commands."""
+        return [str(command) for command in cls]
```

Comment:
as a suggestion, 
we can write dictionary using enum
`{
     [Commands.CHANGE]: { helpMessage: 'your text', 'Change phone number for existed contact.', usageMessage: "Usage: change <name> <phone> <new_phone>"}
  }`

then we can use it in you method. Go by all Commands value. Find in dictionary, show help message + usageMessage. If not found - show "Not instruction provided". If we go through Commands we are sure we show all possible commands to the User. Because after somebody add new command (find_contacts, add_email) - we will not provide it in description list to the User

Reply:
then we can use usageMessage in error decorator, and after we change a dictionary - it will change any place in the app

Reply:
You right, it's a good idea for future improvement